frontend/src/reducers/users.ts

```diff
@@ -1,12 +1,28 @@
 import { Action, ActionTypes, UsersData } from '../actions';
 
+const followUserFunc = (state: UsersData, action: Action) => {
+  let users = state.data || [];
+  let to_follow = users.find((u) => u.id === action.payload);
+
+  return {
+    data: users.map((u) => {
+      if (u.id == to_follow?.id) {
+        return { ...to_follow, following: !to_follow.following };
+      }
+      return u;
```

Comment:
Please use readable variables instead of letter only
```suggestion
    data: users.map((user) => {
      if (user.id == to_follow?.id) {
        return { ...to_follow, following: !to_follow.following };
      }
      return user;
```

Reply:
done